examples/kitchensink/app-counter/package.json

```diff
@@ -4,9 +4,33 @@
   "description": "Kitchensink counter app in Frint",
   "main": "index.js",
   "scripts": {
-    "build": "../../../node_modules/.bin/webpack"
+    "build": "../../../node_modules/.bin/webpack",
```

Comment:
if defining all packages being used as deps already in this file, better not re-use these from root.